lighty-core/lighty-codecs/src/test/java/io/lighty/codecs/DataCodecTest.java

```diff
@@ -128,6 +128,31 @@ public void testConvertNonexistingIdentifier() {
         dataCodec.convertIdentifier("/badToaster:badToaster");
     }
 
+    @Test
```

Comment:
Add negative tests too. For example if parts of String are in wrong order, contains typo somewhere, are not well formatted, etc.